keyboards/framework/config.h

```diff
@@ -0,0 +1,131 @@
+// Copyright 2022 Framework Computer
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// Allow (emulated) EEPROM reset using VIA
+#define VIA_EEPROM_ALLOW_RESET
+
+// Increase to 16K for more storage space
+// With 64K backing size, this spread the wear level over 4 blocks
+#define WEAR_LEVELING_LOGICAL_SIZE (4 * 4096)
+// Increase to 64K backing size to allow more wear leveling
+#define WEAR_LEVELING_BACKING_SIZE (16 * 4096)
+// We use 1MB flash
+#define PICO_FLASH_SIZE_BYTES (1 * 1024 * 1024)
+#define WEAR_LEVELING_RP2040_FLASH_SIZE PICO_FLASH_SIZE_BYTES
+// Keep the last 4K sector free for the serial number
+#define WEAR_LEVELING_RP2040_FLASH_BASE ((PICO_FLASH_SIZE_BYTES) - (WEAR_LEVELING_BACKING_SIZE + 4096))
+
+// Prints every second how many matrix scans were done (Frequency in Hz)
+// Only during debugging, since it prevents the host from going to sleep
+//#define DEBUG_MATRIX_SCAN_RATE
+
+// Prototype for Framework Laptop 16 with Raspberry Pi Pico
+// It'll disable column drive because we need to remap some pins to there
+// SDB is mapped to GP22
+//#define PICO_FL16
+
+#ifdef KEYBOARD_framework_ansi
+#    define LED_CAPS_LOCK_PIN GP24
+#elif KEYBOARD_framework_iso
+#    define LED_CAPS_LOCK_PIN GP24
+#elif KEYBOARD_framework_jis
+#    define LED_CAPS_LOCK_PIN GP24
+#endif
```

Comment:
This could be moved to the info.json for each board/revision.

Reply:
Thanks